### DISCUSSION TITLE
AutoComplete: Send form on Enter key

### DISCUSSION THREAD

**Discussion by lostgeek** [primefaces/community#794] [PrimeVue]
It seems that InputText and AutoComplete have different UX. When you press enter on an InputText, the form is submitted, whereas on AutoComplete (after selecting a suggestion) nothing happens. I think there should at least be an option to send the form on enter press when the overlay is not being displayed.

It might be that the original intent was to have the enter key open up the suggestions again (see [here](https://github.com/primefaces/primevue/blob/master/components/lib/autocomplete/AutoComplete.vue#L619)), but `onArrowDownKey` discards the key press too, when the overlay is not being displayed, resulting in nothing happening.

**Comment by tugcekucukoglu** [comment]
We can expect more user demand for this.

Thank you
